### PR TITLE
fixes: NitkarshChourasia/modules #14

### PR DESCRIPTION
**Pull request by PBJI**
- Update README.md
- Update README.md
- update: Adding a proper README.md file.
- feature: Pseudo database in hash for contributors.
- fix: Correcting a typo.
- update: Adding info on contributions.
- add: contributors.md file.
- add: contributing_guidelines.md file.
- refactor: used descriptive variable names.
- fix: Fixes in uttt.js uttt.test.js and uttt.predict.js

### CODE REVIEW

File: contributors.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+}
+
+// Usage:
+// Use your fullname to create a new contributor object.
+// If unable to use your full name, use your github username instead.
+const TestUser = new Contributor("Test User", "testuser@email.com", "testSuperHero", "https://github.com/testPseudoUrl");
+
+const NitkarshChouarsia = new Contributor("Nitkarsh Chouarsia", "playnitkarsh@gmail.com", "NitkarshChouarsia", "https://github.com/NitkarshChouarsia");
+
+const PurshotamBohra = new Contributor("Purshotam Bohra", "test@gmail.com", "test_username", "test_url");
```
Comment (NitkarshChourasia) [inline contributors.js:36]: Update this.
Always review here.

**Review by NitkarshChourasia — COMMENTED** [review]
Only uttt.js is improved.
Other file dependent on it, yet to be improved.

**Comment by PBJI** [comment]
I refactored the other files in the 8th commit. First I did gh pr checkout 14, then made changes on top of your changes and then created this new pull request.

**Review by NitkarshChourasia — COMMENTED** [review]
Perfect!

**Pull request merged**